src/network/event_loop.rs

```diff
@@ -331,6 +331,9 @@ impl PyrsiaEventLoop {
             SwarmEvent::ConnectionClosed { .. } => {}
             SwarmEvent::OutgoingConnectionError { peer_id, error, .. } => {
                 if let Some(peer_id) = peer_id {
+                    if peer_id == *self.swarm.local_peer_id() {
+                        warn!("The dialed node has the same peer ID as the peer node: '{}', both nodes require unique peer ID.", peer_id);
```

Comment:
`peer node` does not sound like a correct term. Is `current node` or `local node` perhaps better? `dialer node` also came up in my mind, but that might be confusing when having both "dialed" and "dialer" in the same message.

```suggestion
                        warn!("The dialed node has the same peer ID as the current node: '{}'. Please make sure that every node has a unique peer ID.", peer_id);
```

Reply:
@tiainen makes sense, updated.